docs/book/src/tasks/automated-machine-management/healthchecking.md

```diff
@@ -209,15 +209,14 @@ This is useful for dynamically scaling clusters where the number of machines kee
 
 ## Skipping Remediation
 
```

Comment:
I would suggest to further simplify this to
```
There are scenarios where remediation for a machine may be undesirable (eg. during cluster migration using `clusterctl move`). For such cases, MachineHealthCheck skips marking a Machine for remediation if:

- the Machine has the `cluster.x-k8s.io/skip-remediation` annotation
- the Machine has the `cluster.x-k8s.io/paused` annotation
- the MachineHealthCheck has the `cluster.x-k8s.io/paused` annotation
- the Cluster has `.spec.paused` set to `true`
```

Just a lot easier to parse in my opinion

(I didn't find any evidence of handling of the `cluster.x-k8s.io/paused` annotation on clusters, please let me know if I missed it)

Reply:
@sbueringer I agree, this is more readable.
I think `cluster.x-k8s.io/paused` is being handled similar to other annotation.